### ISSUE TITLE
SMAP debug information not installed for Xtend files in 2.11.0.RC1 in xtext.ui

### ISSUE THREAD

**Issue by meysholdt** [eclipse/xtext-eclipse#172]
This causes the Eclipse Debugger to ignore the Xtend source files.

Steps to reproduce:
Install org.eclipse.xtext.ui_2.11.0.v20170117-1027.jar into your target platform.
Install https://marketplace.eclipse.org/content/bytecode-outline into your host Eclipse.

Look at the bytecode from org.eclipse.xtext.ui.generator.EclipseBasedShouldGenerate. Nothing is mentioned about SMAP or Xtend.

In contrast, if you look at the byte code from org.eclipse.xtext.generator.IShouldGenerate, the ByteCode View displays 

```
  // compiled from: IShouldGenerate.java
  // debug info: SMAP
IShouldGenerate.java
Xtend
*S Xtend
*F
+ 0 IShouldGenerate.xtend
org/eclipse/xtext/generator/IShouldGenerate.xtend
*L
25:30
39:40,3
40:43
39:44
41:45
42:46,6
50:59
*E
```

**Comment by cdietrich** [comment]
@meysholdt is this caused by https://github.com/eclipse/xtext-eclipse/commit/66de8b78e7620b42039b31dd87eab3af12d13a01 

? must the smap thing be enabled explicitely?

**Comment by meysholdt** [comment]
I would not say 66de8b78e7620b42039b31dd87eab3af12d13a01 caused this, as it is actually a step towards fixing this issues. Just invoking the Xtend compiler a second time after the Tycho compiler was missing: 09da09d06809b8d7fe03e061a0735f6a6027c3f1

**Comment by cdietrich** [comment]
i assume we need the same in xtext-xtend right?

**Comment by meysholdt** [comment]
Xtend? Maybe. Filed eclipse/xtext-xtend#141

**Comment by spoenemann** [comment]
Fixed with #173.
